docs/content/bundle/manifest/file-format/1.0.0.md

```diff
@@ -146,69 +148,72 @@ status:
+| outputs.type                           | true     | The output type. Allowed values are: string, number, integer, boolean, object, or file.                                                                                                          |
+| outputs.sensitive                      | false    | Indicates whether this output's value is sensitive and should not be logged.                                                                                                                     |
+| images                                 | false    | A map of images referenced in the bundle. When the bundle is relocated, the referenced images are copied to the new bundle location.                                                             |
+| images.KEY.description                 | false    | A description of the image and how it is used.                                                                                                                                                   |
+| images.KEY.repository                  | true     | The repository portion of the image reference, i.e. docker.io/library/nginx                                                                                                                      |
+| images.KEY.digest                      | true     | The repository digest of the image, e.g. sha256:cafebabe...                                                                                                                                      |
+| dependencies                           | false    | Other bundles that this bundle relies upon.                                                                                                                                                      |
+| dependencies.requires                  | true     | A list of bundles that should be executed with this bundle.                                                                                                                                      |  
+| dependencies.requires.name             | true     | A name for the dependency, used to refer to the dependency using the template syntax `bundle.dependencies.NAME`.                                                                                 |
+| dependencies.requires.bundle.reference | true     | The full bundle reference for the dependency in the format REGISTRY/NAME:TAG.                                                                                                                    |
+| dependencies.requires.bundle.version   | false    | [Semver v3 constraint] used to select a more recent release of the bundle. May use wildcard syntax such as `1.0.x` or `2.*`. Use vX.Y.Z-0 to allow prerelease versions to be resolved by Porter. |
```

Comment:
Is there a link to docs about semver v3? Looks like the example bundle has a little more complex constraint than I would understand how to create just from this docs blurb

Reply:
Ah it didn't click that this was a link, that's perfect

Reply:
I've created a follow-up issue for making a separate semver page https://github.com/getporter/porter/issues/2594